Yoast/wordpress-seo
# Fixes the images by running grunt build:images

## Summary


This PR can be summarized in the following changelog entry:

* Fixes the images by running `grunt build:images`.

## Relevant technical choices:

Some differences in the new addon SVGs:
- focusable="false" moved to another spot.
- some points have changed, this might be optimizations?

I could not spot any visible changes between the files.

## Test instructions

This PR can be tested by following these steps:

I generated the following zip with the images: new is this branch, old is trunk.
[images.zip](https://github.com/Yoast/wordpress-seo/files/3831488/images.zip)

Or you can generate them yourself:
* Switch to trunk, make sure there are no changes.
* Copy the affected images, so we can compare.
* Switch to this branch (this is the same as on `trunk` and run `grunt build:images`).

* Compare the images.

## UI changes

## Documentation

## Quality assurance


Fixes #13791
